src/main/java/org/kpmp/slides/Aperio.java

```diff
@@ -26,4 +28,13 @@ public void setOriginalWidth(String OriginalWidth) {
 	}
 	
 
+	public String getMPP() {
```

Comment:
I am not sure this is what you need...the properties are actually:
openslide.mpp-x and openslide.mpp-y and the numbers may be different, and we should get and return each.